pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
```

Comment:
WOW WOW WOW WOW great work bringing the project to the 21 century! Look at you guys using a toml. Such badasses.

Reply:
thank you!